TPlatform.sln

```diff
@@ -61,7 +61,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "WebTests", "WebTests", "{62
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GUITests", "GUITests\GUITests.csproj", "{84259840-2408-413F-9720-67BA90EE98A4}"
 EndProject
```

Comment:
Why do you need to change this?

Reply:
> Why do you need to change this?

No idea, that's some automatic changes

Reply:
Probably have to discard this changes in otrder to not have conflicts with other feature